tests/base.py

```diff
@@ -182,14 +182,12 @@ def setUp(self, config_dict='config_dict'):
                         {
                             'id': '122239',
                             'key': 'control',
-                            'featureEnabled': True,
-                            'variables': [{'id': '155551', 'value': '42.42'}],
+                            'variables': [],
```

Comment:
Removed these since this is not a feature experiment so shouldn't have these props.